docs/requirements.txt

```diff
@@ -0,0 +1,7 @@
+mkdocs-material
+mkdocstrings[python]
+markdown-include
+mkdocs-gen-files
+mkdocs-literate-nav
+mkdocs-section-index
+Black
```

Comment:
What is this ?

Reply:
mkdocs extensions needed for the reference generation i wrote. (by the recipe)